src/main/java/io/lighty/yang/validator/formats/JsonTree.java

```diff
@@ -252,36 +281,49 @@ public Help getHelp() {
         return Optional.empty();
     }
 
-    private JSONObject resolveChildMetadata(final DataSchemaNode node, final Optional<Boolean> isConfig) {
```

Comment:
We can probably invoke stack.enter and stack.exit in the beginning and in the end of this method.
That way we can eliminate need to call enter and exit methods before and after this one.

```
stack.enter()
resolveChildMetadata()
stack.exit
```

becomes to be:

```
resolveChildMetadata()
```

Reply:
Done